.github/workflows/upload-release-artifacts.yaml

```diff
@@ -0,0 +1,74 @@
+name: "upload-release-artifacts"
+
+on:
+  workflow_run:
+    workflows: ["create-release"]
+    types:
+      - completed
+
+jobs:
+  release:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      # Run each build to completion, regardless of if any have failed
+      fail-fast: false
+
+      matrix:
+        os:
+          - ubuntu-20.04
+          - ubuntu-18.04
+          - macOS-10.15
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: install stack (Linux)
+        if: runner.os == 'Linux'
+        run: |
+          curl -L https://github.com/commercialhaskell/stack/releases/download/v2.5.1/stack-2.5.1-linux-x86_64.tar.gz | tar -xz
+          echo "$HOME/stack-2.5.1-linux-x86_64/" >> $GITHUB_PATH
+      - name: install stack (macOS)
+        if: runner.os == 'macOS'
+        run: |
+          curl -L https://github.com/commercialhaskell/stack/releases/download/v2.5.1/stack-2.5.1-osx-x86_64.tar.gz | tar -xz
+          echo "$HOME/stack-2.5.1-osx-x86_64/" >> $GITHUB_PATH
+
+
+      # One of the transcripts fails if the user's git name hasn't been set.
+      - name: set git user info
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"
+
+      - name: remove ~/.stack/setup-exe-cache on macOS
+        if: runner.os == 'macOS'
+        run: rm -rf ~/.stack/setup-exe-cache
+
+      - name: build
+        run: stack --no-terminal build --flag unison-parser-typechecker:optimized
```

Comment:
I think the above is missing `-O2` / `optimized` on some of the other modules.  Should we switch back to the global
```
run: stack --no-terminal build --ghc-options -O2
```
?